observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,31 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6-1_{context}"]
+== Network Observability Operator 1.6.1
+The following advisory is available for the Network Observability Operator 1.6.1:
+
+* link:https://access.redhat.com/errata/RHSA-2024:4785[2024:4785 Network Observability Operator 1.6.1]
+
+[id="network-observability-operator-CVE-1-6-1_{context}"]
+=== CVEs
+* link:https://access.redhat.com/errata/RHSA-2024:4237[RHSA-2024:4237]
+* link:https://access.redhat.com/errata/RHSA-2024:4212[RHSA-2024:4212]
+
+[id="network-observability-operator-1.6.1-bug-fixes_{context}"]
+=== Bug fixes
```

Comment:
I'm not sure whether using the "with this update" phrasing is something that needs to be used without deviation in Release Notes, but just wanted to point out that seemingly none of the bug fixes here use that exact wording. 

Up to you whether you want to change all instances, but I personally feel like at the very least the fixes that use "Now," phrasing should be changed to be more like the fixes that use "with this fix". That's just my opinion though

Reply:
Changed all to With this fix